string      au1SignalName
float64     u8SignalValue